sample-apps/python-sample-app/requirements.txt

```diff
@@ -0,0 +1,51 @@
+backoff==1.11.1
```

Comment:
Can you check the requirements.txt and make sure that we are only adding dependencies that are really ncessary? e.g.: Are we really using the following?

* jinja2
* opentelemetry-instrumentation-jinja2
* urllib3
* ....

Every dependency that is added here and not used is a liability and a source of noise for updates.